### PR TITLE
Remove support for Gazebo version < 7  bug #287

### PR DESCRIPTION
**Pull request by triccyx**

### CODE REVIEW

**Comment by traversaro** [comment]
The Travus failure was spurious.

File: plugins/CMakeLists.txt [hunk 1]
```diff
@@ -25,22 +25,8 @@ else()
...
 endif()
 
-if( GAZEBO_VERSION VERSION_GREATER 3.9 )
-    add_subdirectory(worldinterface)
-else()
-    message(WARNING "Gazebo version is lower than 4.0, disabling worldinterface plugin.")
-endif()
+add_subdirectory(worldinterface)
+add_subdirectory(lasersensor)
+add_subdirectory(depthCamera)
```
Comment (traversaro) [inline plugins/CMakeLists.txt]: Loving it!

Comment (triccyx) [reply plugins/CMakeLists.txt]: So all right?

Comment (traversaro) [reply plugins/CMakeLists.txt]: Yes yes, I was enthusiastic!

**Comment by traversaro** [comment]
Perfect! Can you just target the `devel` branch instead of the `master` one? (This can be changed by the GitHub UI). Thanks!

**Pull request merged**